src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/NetCorePackageReferenceProjectProvider.cs

```diff
@@ -34,11 +37,17 @@ public class NetCorePackageReferenceProjectProvider : INuGetProjectProvider
         public RuntimeTypeHandle ProjectType => typeof(NetCorePackageReferenceProject).TypeHandle;
 
         [ImportingConstructor]
-        public NetCorePackageReferenceProjectProvider(IProjectSystemCache projectSystemCache)
+        public NetCorePackageReferenceProjectProvider(
+            [Import(typeof(SVsServiceProvider))]
+            IServiceProvider vsServiceProvider,
+            IProjectSystemCache projectSystemCache)
         {
             Assumes.Present(projectSystemCache);
```

Comment:
Check that `vsServiceProvider` is not `null`.  I know `AllowDefault=true` is not specified, but a) there's already a `null` check for the other argument, b) if this constructor were to be called by something other than MEF the argument should still be non-`null`, and c) if `vsServiceProvider` were `null`, we'd get a NullReferenceException later in `TryCreateNuGetProject(...)`